src/cli/project.ts

```diff
@@ -0,0 +1,172 @@
+    sindri.logger.debug(
+      `Successfully loaded "sindri.json" from "${sindriJsonPath}":`,
+    );
+    sindri.logger.debug(sindriJson);
+  } catch (error) {
+    sindri.logger.fatal(
+      `Error loading "${sindriJsonPath}", perhaps it is not valid JSON?`,
+    );
+    sindri.logger.error(error);
+    return process.exit(1);
+  }
+  if (!("name" in sindriJson) || typeof sindriJson.name !== "string") {
+    sindri.logger.error('No "name" field found in "sindri.json". Aborting.');
+    return process.exit(1);
+  }
+  return sindriJson.name;
+}
+
+async function getSomeProjectNames(teamName: string): Promise<string[]> {
+  try {
+    const projects = await sindri._client.internal.projectListPaginated({
+      team_name: teamName,
+    });
+    return projects.items.map(({ name }) => name);
+  } catch (error) {
+    if (error instanceof ApiError && error.status === 401) {
+      sindri.logger.error(
+        "Your credentials are invalid. Please log in again with `sindri login`.",
+      );
+    } else {
+      sindri.logger.fatal("An unknown error occurred.");
+      sindri.logger.error(error);
+    }
+    return process.exit(1);
+  }
+}
+
+async function getTeamName(): Promise<string> {
+  try {
+    const response = await sindri._client.internal.teamMe();
+    return response.team.slug;
+  } catch (error) {
+    if (error instanceof ApiError && error.status === 401) {
+      sindri.logger.error(
+        "Your credentials are invalid. Please log in again with `sindri login`.",
+      );
+    } else {
+      sindri.logger.fatal("An unknown error occurred.");
+      sindri.logger.error(error);
+    }
+    return process.exit(1);
+  }
+}
+
+const projectDeleteCommand = new Command()
+  .name("delete")
+  .description("Delete a project and all associated builds and proofs.")
+  .option(
+    "-a, --all",
+    "Delete *all* projects. Overrides the `--name` and directory options.",
+  )
+  .option("-f, --force", "Skip the confirmation prompt.")
+  .option(
+    "-n, --name <name>",
+    "The name of the project to delete. Overrides the directory option.",
+  )
+  .argument("[directory]", "The location of the Sindri project to delete.", ".")
+  .action(async (directory, { all, force, name }) => {
+    // Check that the API client is authorized.
+    if (!sindri.apiKey || !sindri.baseUrl) {
+      sindri.logger.warn("You must login first with `sindri login`.");
+      return process.exit(1);
+    }
+
+    // Ask the user for confirmation.
+    if (!force) {
+      const userConfirmed = await confirm({
+        message:
+          "This will permanently delete the project(s) and any associated proofs. Proceed?",
+        default: true,
+      });
+      if (!userConfirmed) {
+        sindri.logger.warn("Aborting. No projects were deleted.");
+        return;
+      }
+    }
+
+    // Perform the deletion.
+    let deletedCount = 0;
+    if (!all) {
+      name = name || findProjectName(directory);
+      await deleteProject(name);
```

Comment:
Off by one in this case (so the log would say "...deleted 0 projects")
```suggestion
      await deleteProject(name);
      deletedCount++;
```